homework2/reviewapp/src/app/components/features/shows/ReviewForm/ReviewForm.tsx

```diff
@@ -0,0 +1,34 @@
+import { IReview } from "@/app/typings/review";
+import { Button, Flex, Input } from "@chakra-ui/react";
+
+export interface IReviewFormProps
+{
+    onAddReview:(review:IReview)=>void;
+}
+export const ReviewForm=({onAddReview}:IReviewFormProps) =>
```

Comment:
Since you did name the component `ReviewForm` it would be a good idea to use a form element here and handle this more in "form" sense.
We can go over this in consultations.